### PR TITLE
cook mig chat

### PR DESCRIPTION
**Pull request by Kageschwump**

### CODE REVIEW

**Comment by Hamsterpants** [comment]
LGTM medium rare

File: report.md [hunk 1]
```diff
@@ -55,11 +55,53 @@
...
+- **LandingPage**: The first page of the application that the user will see.
+- **PokerPage**: A page for the poker game.
+- **CoinflipPage**: A page for the coin flip game.
+- **BlackjackPage**: A page for the blackjack game.
+
+These components are just empty at this point, but functionality will be developed further as the project progresses.
+
+#### TypeScript Integration
+
+The frontend is also built using **TypeScript**, which enables typing. This is useful to help catch errors during development rather than at runtime. TypeScript is best used by defining **interfaces** and **types** for components’ props and states. This ensures that data passed between components is structured correctly.
```
Comment (muItilingualism) [inline report.md:91]: "...which enables typing." -> "...which enables static type checking."

File: report.md [hunk 2]
```diff
@@ -55,11 +55,53 @@
...
+
+[EXAMPLES]
```
Comment (muItilingualism) [inline report.md:93]: missing examples?

File: report.md [hunk 3]
```diff
@@ -55,11 +55,53 @@
...
+
+#### Styling with Tailwind CSS
+
+For the styling of the front end, **Tailwind CSS** is used. Tailwind CSS is one of many CSS frameworks that makes it easier to style different components. One of the advantages of Tailwind is that it has built-in responsiveness, making it simple to create a layout that adapts to different screen sizes.
```
Comment (muItilingualism) [inline report.md:97]: "For the styling of the frontend, ..." -> "To style the frontend, ..."

File: report.md [hunk 4]
```diff
@@ -55,11 +55,53 @@
...
+
+#### Build Tool: Vite
+
+As for building the application, we use **Vite**. Vite is a build tool that focuses on the developer experience by enabling hot-reloading, which improves the experience for developers when working on the application.
```
Comment (muItilingualism) [inline report.md:101]: "... use Vite. Vite is..." cringe 

Vite at the end and start of the next sentence. please reword :)

File: report.md [hunk 5]
```diff
@@ -55,11 +55,53 @@
...
+
+[EXAMPLES]
```
Comment (muItilingualism) [inline report.md:103]: Missing examples?

**Review by muItilingualism — CHANGES_REQUESTED** [review]
Minor changes.

Where are the examples?

**Review by HenrikSkindhoj — APPROVED** [review]
Smukt :)

**Pull request merged**